scripts/add_cluster_resource.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+_BN="$(basename "${0}")"
+
+_usage() {
+cat <<HEREDOC
+
+Fetch a cluster resource from github and add it to cluster-scope app.
+This script requires the user to be logged into an existing OCP cluster
+that supports the api resources meaning to be added.
+
+Usage:
+  ${_BN} [env] [cluster] [flags]
+
+Mandatory flags (mutually exclusive):
+  -f, --file    path to file on local filesystem
+  -u, --url     remote git url for the raw file
+
+Optional Flags:
+  -h, --help    help for this script
+
+Example:
+  ${_BN} moc zero -f /tmp/manifest.yaml
+  ${_BN} moc zero -u https://raw.githubusercontent.com/<github_user>/<repo>/master/<path-to-manifest>/manifest.yaml
+
+HEREDOC
+}
+
+# Keep values sorted alphabetically since `kustomize edit resource add` just appends.
+_sort_kustomize_values(){
+  tmpfile=$(mktemp)
+  yq e -j kustomization.yaml | jq 'map_values(select(. >= []) |= sort)' | yq e -P > ${tmpfile}
+  mv ${tmpfile} ./kustomization.yaml
+  kustomize edit fix
+}
+
+_get_plural(){
+  kind=${1}
+  apiVersion=${2}
+  kubectl api-resources  | grep "\b${kind}\b" | grep ${apiVersion} | awk '{print $1}'
```

Comment:
Got a little lazy here, since I didn't want to add a lot of logic for finding plurals for resources that end with words like `policy -> policies`. Open to suggestions.